docs/cli/overview.md

```diff
@@ -0,0 +1,54 @@
+---
+title: .NET Aspire CLI Overview and Commands
+description: Learn .NET Aspire CLI commands for creating projects, running an app host, and adding integrations. Get started with command-line tools to build and manage distributed applications efficiently.
+ms.date: 06/26/2025
+ms.topic: overview
+ms.custom:
+  - ai-gen-docs-bap
+  - ai-gen-title
+  - ai-seo-date:06/26/2025
+  - ai-gen-description
+---
+
+# .NET Aspire CLI Overview
+
+// TODO: More selling of why you want to use this instead of the .NET CLI you're probably already familiar with.
+
+The .NET Aspire CLI (`aspire` command) is a .NET global tool that provides command-line functionality to create, manage, run, and publish Aspire projects. Use the .NET Aspire CLI to streamline development workflows for distributed applications.
+
+- [Install .NET Aspire CLI](install.md)
+
+## Create projects
+
+The `aspire new` command creates one or more Aspire projects  that the Aspire CLI tool installs the latest Aspire project templates into the `dotnet` system.
+
+Use the `aspire new` command to create an Aspire project from a list of templates. Once a template is selected, the name of the project is set, and the output folder is chosen, `aspire` downloads the latest templates and generates one or more projects.
+
+## Run the app host project
+
+// TODO: This also checks certs, and does it do anything else? What else should we point out?
```

Comment:
In order (sourced from [here](https://github.com/dotnet/aspire/blob/main/src/Aspire.Cli/Commands/RunCommand.cs#L60)):

- Locates app host project.
- Checks certificates.
- Trusts certificates.
- Builds app host project.
- Verifies app host compatibility.
- Runs the app host:
  - Displays link to launched dashboard.
  - Displays table of resources with endpoints.